test/utility/bc-google-analytics.spec.js

```diff
@@ -154,4 +156,99 @@ describe('Baltimore County Google Analytics Utility', () => {
 			expect(actual).toEqual(true)
 		});
 	});
+
+	describe('getDefaultEvent', () => {
+		it('should return null if not a default action', () => {
+			const mockEvent = {
+				action: 'contact',
+				category: 'Branch Locations',
+				label: 'Arbutus'
+			};
+			const actual = googleAnalytics.getDefaultEvent(mockEvent);
+
+			expect(actual).toBeNull();
+		});
+		
+		it('should return an object with the key "method" and the value "Polaris"', () => {
+			const expected = {
+				method: 'Polaris'
+			};
+			const actual = googleAnalytics.getDefaultEvent({
+				action: 'login',
+				label: 'Polaris'
+			});
+
+			expect(actual).toEqual(expected);
+		});
+
+		it('should return an object with the key "method" and the value "Twitter"', () => {
+			const expected = {
+				method: 'Twitter'
+			};
+			const actual = googleAnalytics.getDefaultEvent({
+				action: 'share',
+				label: 'Twitter'
+			});
+
+			expect(actual).toEqual(expected);
+		});
+
+		it('should return an object with the key "search_term" and the value "book"', () => {
+			const expected = {
+				search_term: 'book'
+			};
+			const actual = googleAnalytics.getDefaultEvent({
+				action: 'search',
+				label: 'book'
+			});
+
+			expect(actual).toEqual(expected);
+		});
+
+		it('should return an object with the key "search_term" and the value "book"', () => {
+			const expected = {
+				search_term: 'book'
+			};
+			const actual = googleAnalytics.getDefaultEvent({
+				action: 'view_search_results',
+				label: 'book'
+			});
+
+			expect(actual).toEqual(expected);
+		});
```

Comment:
These all looks great, i would check the other two default events in addition to these.

Reply:
I think I have them all. `login` / `share` both return the key `method` and `search` / `view_search_results` return the key `search_term`.

Reply:
yep, I'm dumb, disregard please